data/templates/load-balancing/haproxy.cfg.j2

```diff
@@ -69,6 +69,16 @@ frontend {{ front }}
 {%         endif %}
 {%         if front_config.mode is vyos_defined %}
     mode {{ front_config.mode }}
+{# add tcp-request related directive if ssl is configed #}
+{%             if front_config.mode is vyos_defined('tcp') and front_config.rule is vyos_defined %}
+{%                 for rule, rule_config in front_config.rule.items() %}
+{%                     if rule_config.ssl is vyos_defined %}
+    tcp-request inspect-delay 5s
```

Comment:
Hardcode a value is not a good idea.

Please move this up to the CLI and rather specify `<defaultValue>5</defaultValue>` thus there will be a proper default but the user knows it's value due to completion help and also can adjust it.

Reply:
changed the design to add a `tcp-request` node in `interface-definitions/include/haproxy/tcp-request.xml.i`, then invoke this node in `interface-definitions/load-balancing_reverse-proxy.xml.in`, please check the latest Files changed